app/src/main/java/org/mozilla/reference/browser/browser/BrowserFragment.kt

```diff
@@ -24,7 +24,7 @@ import org.mozilla.reference.browser.tabs.TabsTrayFragment
  * Fragment used for browsing the web within the main app.
  */
 class BrowserFragment : BaseBrowserFragment(), UserInteractionHandler {
-    private val thumbnailsFeature = ViewBoundFeatureWrapper<ThumbnailsFeature>()
+    private val thumbnailsFeature = ViewBoundFeatureWrapper<BrowserThumbnails>()
```

Comment:
We should revisit how this component is used as we iterate further. We don't usually use a browser component directly (they connect with other bits and wrapped in a feature).